widgets/contact/contact.php

```diff
@@ -846,13 +852,18 @@ function contact_form_action( $instance, $storage_hash ){
 			}
 		}
 
+		// Add in the default sender if no email field is defined 
+		if ( ! isset( $email_fields['email'] ) && ! empty( $instance['settings']['default_sender'] ) ) {
+			$email_fields['email'] = $instance['settings']['default_sender'];
```

Comment:
Perhaps here, if `email` and `default_sender` is empty, populate the `email` field with the current user's email address. Similar to what we're doing on [L557-558](https://github.com/dl4ner/so-widgets-bundle/blob/a8240aed0a11b9e0b4745917158f4c609a4fbae4/widgets/contact/contact.php#L557-L558)